html/browsers/history/the-location-interface/location-stringifier.html

```diff
@@ -8,4 +8,16 @@
 <div id=log></div>
 <script>
 test_stringifier_attribute(location, "href", true);
+
+test(function() {
+  const prop1 = Object.getOwnPropertyDescriptor(location, "toString"),
+        prop2 = Object.getOwnPropertyDescriptor(location, "href")
+
+  assert_equals(prop1.configurable, prop2.configurable)
```

Comment:
I think this would be clearer if you just tested prop2's properties directly with assert_true/assert_false. Also it'd be good to test that `typeof prop2.get === "function"`.